alf/algorithms/trac_algorithm.py

```diff
@@ -68,15 +69,15 @@ def __init__(self,
 
         Args:
             action_spec (nested BoundedTensorSpec): representing the actions.
-            ac_algorithm_cls (type): Actor Critic Algorithm cls (AC, PPO, SAC...)
+            ac_algorithm_cls (type): Actor Critic Algorithm cls.
             action_dist_clip_per_dim (float): action dist clip per dimension
             debug_summaries (bool): True if debug summaries should be created.
             name (str): Name of this algorithm.
         """
         ac_algorithm = ac_algorithm_cls(
             action_spec=action_spec, debug_summaries=debug_summaries)
 
-        assert isinstance(ac_algorithm._actor_network, DistributionNetwork)
+        assert hasattr(ac_algorithm, '_actor_network')
```

Comment:
TracAlgorithm does not use ac_algorithm._actor_network.

Reply:
`TrustedUpdater` update parameters for  `ac_algorithm._actor_network`  (L106)